static/js/manage/backlog-overview.js

```diff
@@ -1,4 +1,60 @@
 MEDUSA.manage.backlogOverview = function() {
+
+    $(document).ready(function() {
```

Comment:
Remove the document ready as this whole function is run when the page starts so it should work just by using the `checkManualSearches();`.